python/orion_finance_sdk_py/stats/series.py

```
@@ -0,0 +1,122 @@
+    """
+
+    def __init__(
+        self,
+        returns: pd.DataFrame,
+        *,
+        prices: pd.DataFrame | None = None,
+        periods_per_year: int = DEFAULT_PERIODS_PER_YEAR,
+    ) -> None:
+        """Store already-hygiened contiguous daily returns and optional prices."""
+        if periods_per_year <= 0:
+            raise ValueError("periods_per_year must be positive")
```

Ruff (0.16.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)